### DISCUSSION TITLE
Weird overlay on browser

### DISCUSSION THREAD

**Discussion by rubiin** [prasanthrangan/hyprdots#1541] [General]
![image](https://github.com/prasanthrangan/hyprdots/assets/8222059/88ee07a7-1831-4890-96a5-c84932a653f6)

![image](https://github.com/prasanthrangan/hyprdots/assets/8222059/77f9acb1-d92c-4be5-af72-8c157551aa5f)

**Comment by kRHYME7** [comment]
running on wayland?

**Reply by rubiin** [reply]
yes

**Reply by kRHYME7** [reply]
what browser?

**Reply by rubiin** [reply]
![image](https://github.com/prasanthrangan/hyprdots/assets/8222059/6c12bf80-433b-42f6-992a-36fac82dac41)


The browser is vivaldi

**Reply by kRHYME7** [reply]
I can only replicate this while vivaldi is in xwayland

can you provide ` hyprctl clients ` 
 
In my case the issue is fixed when I enable wayland for vivaldi 


![image](https://github.com/prasanthrangan/hyprdots/assets/53417443/e8755f97-6a88-4882-977e-6bfc06ff7fb2)


![image](https://github.com/prasanthrangan/hyprdots/assets/53417443/84889573-148b-4c73-9686-0eba61649d8d)

![image](https://github.com/prasanthrangan/hyprdots/assets/53417443/9ab747cb-7428-431e-ae41-fd22e14b7020)
![image](https://github.com/prasanthrangan/hyprdots/assets/53417443/a4d5297b-a25d-4535-9744-5a521db066db)

**Reply by rubiin** [reply]
Looks like that fixed on my case too

**Reply by rubiin** [reply]
Another issue I have been having is when I add the pokemon cli on myzshrc, I get 
![image](https://github.com/prasanthrangan/hyprdots/assets/8222059/7d19bf7c-0e79-4e04-8b1c-0c51e9f8e829)

**Reply by kRHYME7** [reply]
` pokemon-colorscripts --no-title -r 1,3,6 ` 

exactly, be sure to add it in the bottom.  try to add` sleep incase there's a race conditions

**Reply by rubiin** [reply]
![image](https://github.com/prasanthrangan/hyprdots/assets/8222059/a7b20448-f8a6-4b2d-92b4-fe111e30c973)
on bottom

**Reply by kRHYME7** [reply]
I forgot about this but I set this using the quiet flag. You need to enable instant prompt 

https://github.com/romkatv/powerlevel10k?tab=readme-ov-file#instant-prompt

In my case I took the oppurtuinity to re configure my prompt. The 2nd to the last question will be like this, choose 2 to enable instant prompt. 
![image](https://github.com/prasanthrangan/hyprdots/assets/53417443/3f233b97-76c2-4af3-98c1-dffe6557452a)

**Reply by rubiin** [reply]
.p10k from the repo has that set to verbose dunno why
https://github.com/prasanthrangan/hyprdots/blob/1bb2eb17b33284603c5cd362d724b394a6d41744/Configs/.p10k.zsh#L1809

**Reply by kRHYME7** [reply]
![image](https://github.com/prasanthrangan/hyprdots/assets/53417443/8ed9d4a2-87d1-49c0-9ea2-fabc93b7aafd)

I think this is an accident, you can push a MR.

**Reply by rubiin** [reply]
so didnt want to modify it as it was shipped

**Reply by rubiin** [reply]
sure thing

**Reply by kRHYME7** [reply]
So this is one of the reason why zsh is hanging on my VM then.

**Reply by rubiin** [reply]
https://github.com/prasanthrangan/hyprdots/pull/1542
done

**Reply by rubiin** [reply]
@kRHYME7 do you know how to cycle between windows on grouped layout?
`Super + G	Toggle the window between focus and group`

**Reply by rubiin** [reply]
more context https://wiki.hyprland.org/Configuring/Dispatchers/#grouped-tabbed-windows

**Reply by rubiin** [reply]
```
for forward and backward. I think someone should add this to keymaps . 

bind = SUPER+ALT, J, changegroupactive, f
bind = SUPER+ALT, K, changegroupactive, b

``

**Reply by rubiin** [reply]
@kRHYME7 can you suggest me some keybinds that are not used for these two keybinds.

**Reply by kRHYME7** [reply]
It's too user specific but I use this 

![image](https://github.com/prasanthrangan/hyprdots/assets/53417443/e781959b-1239-444f-991d-47fff3431570)

This is user specific thing, so I did not mind adding to the keybindings

**Reply by rubiin** [reply]
I made a PR. Found a key combo that didnt conflict with any
https://github.com/prasanthrangan/hyprdots/pull/1547
